src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -610,6 +645,18 @@ protected override void HandleInitializeRequestAsync(IRequestResponder<Initializ
                 });
             }
 
+            List<ColumnDescriptor> additionalModuleColumns = null;
+            if (responder.Arguments.ClientID == "visualstudio")
```

Comment:
>") [](start = 61, length = 2)

You want to also do it for `liveshare-server-host`

Reply:
BTW: If we have any other code which is checking for a ClientID of 'visualstudio', it needs the same fix.

---
In reply to: [504884955](https://github.com/microsoft/MIEngine/pull/1054#discussion_r504884955) [](ancestors = 504884955)

Reply:
this should be the first time ClientID is used. 

Other instances try to differentiate VS/VSCode with DebugEngineHost, but now that DAP works for VS Code/VS/LiveShare we will need to keep track of ClientID.

Reply:
Correct - this is the first time ClientID is used in MIEngine.